src/test/url-loader/url-resolver_test.ts

```diff
@@ -25,8 +25,10 @@ class SimplestUrlResolver extends UrlResolver {
     return this.simpleUrlResolve(baseUrl, url);
   }
 
+  relative(to: ResolvedUrl): PackageRelativeUrl;
+  relative(from: ResolvedUrl, to: ResolvedUrl, kind?: string): FileRelativeUrl;
```

Comment:
I think for implementations you can leave off the two type lines

Reply:
For some reason it doesn't like it.

```
src/test/url-loader/url-resolver_test.ts(20,7): error TS2415: Class 'SimplestUrlResolver' incorrectly extends base class 'UrlResolver'.
  Types of property 'relative' are incompatible.
    Type '(fromOrTo: ResolvedUrl, maybeTo?: ResolvedUrl | undefined, _kind?: string | undefined) =>FileRel...' is not assignable to type '{ (to: ResolvedUrl): PackageRelativeUrl; (from: ResolvedUrl,to: ResolvedUrl, kind?: string | und...'.
      Type 'FileRelativeUrl | PackageRelativeUrl' is not assignable to type 'PackageRelativeUrl'.
        Type 'FileRelativeUrl' is not assignable to type 'PackageRelativeUrl'.
          Type 'FileRelativeUrl' is not assignable to type 'PackageRelativeUrlBrand'.
            Property 'PackageRelativeUrlBrand' is missing in type 'string & FileRelativeUrlBrand'.

20 class SimplestUrlResolver extends UrlResolver {
         ~~~~~~~~~~~~~~~~~~~
```

Reply:
Hm, but `resolve` is written this way. Oh, maybe it's because they vary based on return type? Ok, yeah, I guess this is the simplest way to write this.